### ISSUE TITLE
global colors do not always work and cause design errors

### ISSUE THREAD

**Issue by SebastianNetgrade** [elementor/elementor#12303]
![Bildschirmfoto 2020-08-28 um 16 43 53](https://user-images.githubusercontent.com/62643978/91581158-63d75a00-e94e-11ea-8b03-2418331e8166.png)

Elementor Version: 3.0.3
Elementor Pro Version: 3.0.1

as you can see in the screenshot, global color settings don't always take effect, or take effect first and change themselves later. the same happens with global font sizes. this error can appear not only in elementor editor, but also in the frontend. sometimes it happens that the text of a whole page has the color red, although a global black was chosen. 

a quick solution so far is to change the color of the element, save it and shortly afterwards change it back to the desired global color and save it again.

in most cases it did not help to empty the cache or regenerate template and CSS files.

i: no caching plugins active.

**Comment by maurocarreraok** [comment]
+1

**Comment by felipepitol** [comment]
+1

**Comment by dafnecd** [comment]
+1

**Comment by whokneadsdough** [comment]
I uploaded a screen recording [https://youtu.be/0HNljn7uoUE](https://youtu.be/0HNljn7uoUE) of many strange behaviors of Global Colors and Fonts, time stamps are below.
(Installation details: Elementor 3.0.3, Elementor Pro 3.0.1, Theme: Hello)

Global Colors and Fonts, Site Settings Strange Behaviors Time Stamps
- 1:20 - Colors and fonts not displaying on live site.
- 1:53 - H1 font-size can be changed, but H2 cannot.
- 3:05 - Changing H3 to a Global Color causes all typography to lose colors and font family settings.
- 3:45 - Changing H1 font-size causes H2 and H3 to inherit secondary Global Color.
- 4:18 - Changing H2 font-size causes H2 and H3 to lose Global Color.
- 4:41 - Changing H1 font-size causes H2 and H3 to re-inherit Global Color.
- 5:45 - After modifying Global Colors, typography settings are lost.
- 5:50 - Global Colors magically appear in typography settings, but are not being applied in the editor.
- 5:57 - Changing Body Text Color to a different Global Color causes the editor to display Global Colors correctly. But some headings and body text revert to a serif font family never specified by user.
- 6:40 - Changing Body Typography to Global Fonts Text causes colors to be lost, font-families change from sans-serif to serif.
- 7:04 - Clicking on the Font-Size field causes font-family to change.
- 8:47 - All previous Typography settings spontaneously.
- 8:51 - Typography settings magically appearing.
- 9:11 - Clicking Body Typography causes editor to display Global Fonts settings.
- 9:30 - Live page shows the Global Fonts and Colors settings are in the CSS, but Chrome is choosing to render something else.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.4

Feel free to update

Thanks!

**Comment by Eazi-G** [comment]
Last week on Friday I updated to WP version 5.7 and proceeded to update all my plugins, the first thing I noticed is that I photos started disappearing after I uploaded them on a page. As from yesterday my global colors switched back and forth without me doing anything. I able to make them appear by regenerating files but they soon go back black after a few hours. Can anyone help? I am begining to be frustrated
